json-schema/schema.json

```diff
@@ -111,12 +109,14 @@
         "s2:generation_time": {
           "title": "Generation Time",
           "type": "string",
-          "format": "date-time"
+          "format": "date-time",
+          "deprecated": true
```

Comment:
deprecate or just remove?

Reply:
Deprecation would be consistent with the other changes. If we remove it would also fail validation for any catalog that still has it.